setup.py

```diff
@@ -59,6 +56,7 @@
     'sphinx_rtd_theme>=0.2.4,<0.5',
     'docutils>=0.14,<0.18',
     'autodocsumm>=0.1.10',
+    'Jinja2<3.1',
```

Comment:
Can you check which Jinja2 version is installed?  What about using this range `Jinja2>=2,<3`?

Reply:
Version 3.1 breaks sphinx, your suggestion works!